### PR TITLE
64 fix GitHub actions for oasis

### PR DESCRIPTION
**Pull request by gabriel0003**
## Proposed changes

Describe the big picture of your changes here.  If it fixes a bug or resolves a feature request, be sure to link that issue!

## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

## Screenshots or links to design files

Does this change impact front end code?

- [x] No
- [ ] Yes (please include screenshots or Figma files)

## Checklist

- [x] Code compiles correctly
- [ ] I have added tests that prove my fix is effective, or that my feature works
- [ ] Unit tests pass locally with my changes
- [ ] I have added/updated all necessary documentation
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+ij_yaml_indent_sequence_value = true
+ij_yaml_keep_indents_on_empty_lines = false
+ij_yaml_keep_line_breaks = true
+ij_yaml_sequence_on_new_line = false
+ij_yaml_space_before_colon = false
+ij_yaml_spaces_within_braces = true
+ij_yaml_spaces_within_brackets = true
+
+[Makefile]
+indent_style = tab
```
Comment (jontyk) [inline .editorconfig:492]: This probably should be removed and added to .gitignore?

Comment (gabriel0003) [reply .editorconfig:492]: https://editorconfig.org
editorconfig has nothing to do with .gitignore. It's about code formatting

**Review by edifirut — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (tsearle) [reply .editorconfig:492]: I agree with Gaby.  This was the way we agreed to enforce a consistent coding style

Comment (jontyk) [reply .editorconfig:492]: Ignore me! Thought it was specific to Gabi's IDE rather than cross setup. The more I know :)

**Pull request merged**